activities.json

```diff
@@ -743,6 +743,18 @@
     "title": "Web MIDI API",
     "url": "https://webaudio.github.io/web-midi-api"
   },
+  {
+    "ciuName": null,
+    "description": "Near Field Communication (NFC) enables wireless communication between two devices at close proximity, usually less than a few centimeters. This document defines an API to enable selected use cases based on NFC technology. The current scope of this specification is NDEF. Low-level I/O operations (e.g. ISO-DEP, NFC-A/B, NFC-F) and Host-based Card Emulation (HCE) are not supported within the current scope.",
+    "id": "web-nfc",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1308614",
+    "mozPosition": "harmful",
+    "mozPositionDetail": "We believe WebNFC poses risks to users security and privacy because of both the wide range of functionality of the existing NFC devices on which it would be supported, and because of the difficulty of meaningfully asking the user for permission to share or write data when the browser cannot explain to the user what is being shared or written.",
```

Comment:
```suggestion
    "mozPositionDetail": "We believe WebNFC poses risks to users security and privacy because of the wide range of functionality of the existing NFC devices on which it would be supported, because there is no system for ensuring that private information is not accidentally exposed other than relying on user consent, and because of the difficulty of meaningfully asking the user for permission to share or write data when the browser cannot explain to the user what is being shared or written.",
```

Reply:
`s/WebNFC/Web NFC/g`

Reply:
How is `there is no system for ensuring that private information is not accidentally exposed other than relying on user consent` different than what is currently happening with camera and microphone access?

Reply:
Can you be more descriptive about the "wide range of functionality" as it seems quite vague?

Reply:
> 

How is "there is no system for ensuring that private information is not accidentally exposed other than relying on user consent" different than what is currently happening with camera and microphone access?

It isn't, except that the final point isn't true in that case, and it's the combination of them that's problematic.